src/foundry/client/data/documents/scene.d.ts

```diff
@@ -122,332 +91,385 @@ declare global {
+    protected _onActivate(active: boolean): Promise<Scene>;
 
-    override _preCreateEmbeddedDocuments(
+    /** @internal */
+    protected override _preCreateEmbeddedDocuments(
       embeddedName: string,
-      result: DrawingDataConstructorData[],
+      result: BaseDrawing.ConstructorData[],
       options: DocumentModificationOptions,
       userId: string
     ): void;
-    _preCreateEmbeddedDocuments(
+    /** @internal */
+    protected override _preCreateEmbeddedDocuments(
       embeddedName: string,
-      result: TokenDataConstructorData[],
+      result: BaseToken.ConstructorData[],
       options: DocumentModificationOptions,
       userId: string
     ): void;
-    _preCreateEmbeddedDocuments(
+    /** @internal */
+    protected override _preCreateEmbeddedDocuments(
       embeddedName: string,
-      result: AmbientLightDataConstructorData[],
+      result: BaseAmbientLight.ConstructorData[],
       options: DocumentModificationOptions,
       userId: string
     ): void;
-    _preCreateEmbeddedDocuments(
+    /** @internal */
+    protected override _preCreateEmbeddedDocuments(
       embeddedName: string,
-      result: NoteDataConstructorData[],
+      result: BaseNote.ConstructorData[],
       options: DocumentModificationOptions,
       userId: string
     ): void;
-    _preCreateEmbeddedDocuments(
+    /** @internal */
+    protected override _preCreateEmbeddedDocuments(
       embeddedName: string,
-      result: AmbientSoundDataConstructorData[],
+      result: BaseAmbientSound.ConstructorData[],
       options: DocumentModificationOptions,
       userId: string
     ): void;
-    _preCreateEmbeddedDocuments(
+    /** @internal */
+    protected override _preCreateEmbeddedDocuments(
       embeddedName: string,
-      result: MeasuredTemplateDataConstructorData[],
+      result: BaseMeasuredTemplate.ConstructorData[],
       options: DocumentModificationOptions,
       userId: string
     ): void;
-    _preCreateEmbeddedDocuments(
+    /** @internal */
+    protected override _preCreateEmbeddedDocuments(
       embeddedName: string,
-      result: TileDataConstructorData[],
+      result: BaseTile.ConstructorData[],
       options: DocumentModificationOptions,
       userId: string
     ): void;
-    _preCreateEmbeddedDocuments(
+    /** @internal */
+    protected override _preCreateEmbeddedDocuments(
       embeddedName: string,
-      result: WallDataConstructorData[],
+      result: BaseWall.ConstructorData[],
       options: DocumentModificationOptions,
       userId: string
     ): void;
 
-    override _onCreateEmbeddedDocuments(
+    /** @internal */
+    protected override _onCreateEmbeddedDocuments(
       embeddedName: string,
       documents: InstanceType<ConfiguredDocumentClass<typeof DrawingDocument>>[],
-      result: DeepPartial<DrawingDataSource>[],
+      result: DeepPartial<BaseDrawing.ConstructorData>[],
```

Comment:
I think we can actually leave out the `DeepPartial` around the `ConstructorData` and `UpdateData`. Their properties (even nested) should all already be optional.

Reply:
Updated all